src/wymeditor/iframe/default/wymiframe.html

```diff
@@ -21,5 +21,5 @@
 <title>WYMeditor iframe</title>
 <link rel="stylesheet" type="text/css" media="screen" href="wymiframe.css" />
 </head>
-<body class="wym_iframe"></body>
+<body></body>
```

Comment:
Gotcha. I wasn't sure if the presence of that class would break anything, but it sounds like it doesn't. :+1: